easybuild/easyconfigs/m/MATLAB/README.md

```diff
@@ -0,0 +1,32 @@
+This example uses Matlab 2018a and MATLAB-2018a.eb  
+Steps to install Matlab:
+1. Install the appropriate Java install which requires manual configuration. See Java documentation for install.
+1. Matlab needs a "File Installation Key". There are a few options available:
+   1. Your Matlab license should have a "File Installation Key" in under "Advanced Options" in the "Install and Activate" tab of your "License Center". This will only install the products associated with this license.
+   1. If you need to install all of the toolboxes to support multiple licenses Mathworks can enable a "All Product File Installation Key". This will install all of the client products but will not install server products like MDCS. A second manual installation on top of the first will be required.
+   1. If you have a server product key, like MDCS, the "File Installation Key" may not install client products like Matlab Compilers. A second manual installation on top of the first will be required.
```

Comment:
Could the section about a requirement for a 'second manual installation' be clarified?

Reply:
More specifically, the latest matlab easyblock can take multiple file installation keys thus allowing you to install the base matlab and MDCS in a single install.